cirq/ops/common_channels.py

```diff
@@ -59,6 +59,7 @@ def __init__(self, p_x: float, p_y: float, p_z: float) -> None:
         self._p_z = value.validate_probability(p_z, 'p_z')
         self._p_i = 1 - value.validate_probability(p_x + p_y + p_z,
                                                    'p_x + p_y + p_z')
+        protocols.validate_mixture(self._mixture_())
```

Comment:
I don't think we should be validating the mixture every time a user constructs a class. The class is supposed to guarantee this property based on validating the probabilities. Verifying that implication is the purpose of unit tests.